src/components/ContestTile.js

```diff
@@ -55,27 +54,30 @@ const ContestTile = ({ contest: { node }, updateContestStatus }) => {
           />
         ) : (
           <p>
-            Contest ran {t.startDay}-{t.endDay}
+            Contest ran {t.startDay === "" ? "" : t.startDay}-
+            {t.endDay === "" ? "" : t.endDay}
           </p>
         )}
         <ClientOnly>
           <Link
-            to={contestPath}
+            to={fields?.contestPath || "/"}
```

Comment:
Similar context/semantics comment here -- `fields.contestPath` is practically mandatory.. and should it link to the main page when it doesn't exist? Seems like there'd be a broader issue.

Reply:
Absolutely, that was a temp fix, don't remember exactly why but I had some errors, now that after re-test it seems good ! :)